docs/kafka/kafka-instance-settings/README.adoc

```diff
@@ -0,0 +1,152 @@
+--
+
+
+[id="proc-editing-kafka-instance-settings_{context}"]
+== Reviewing and editing Kafka instance settings in {product-kafka}
+
+[role="_abstract"]
+Use the {product-long-kafka} web console to review and adjust the Kafka instance settings as needed.
+
+As an alternative to using the {product-kafka} web console, you can use the `rhoas` command-line interface (CLI) to update certain Kafka instance settings, as shown in the following example command:
+
+.Example CLI command to disable connection reauthentication
+[source]
+----
+rhoas kafka update --reauthentication false
+----
+
+For a list of Kafka instance settings that you can update using the CLI, see the `rhoas kafka update` entry in the {base-url-cli}{command-ref-url-cli}[CLI command reference (rhoas)^].
+
+.Prerequisites
+* You have created a Kafka instance. To learn how to do this, see {base-url}{getting-started-url-kafka}[Getting started with {product-long-kafka}^].
+
+.Procedure
+. In the {product-kafka} {service-url-kafka}[web console^], click *Kafka Instances* and select a Kafka instance.
+. Select the *Settings* tab.
+. Adjust any of the settings as needed.
+
+[role="_additional-resources"]
+.Additional resources
+* {base-url}{getting-started-url-kafka}[Getting started with {product-long-kafka}^]
+* {base-url}{getting-started-rhoas-cli-url-kafka}[Getting started with the rhoas CLI for OpenShift Streams for Apache Kafka^]
+* {base-url-cli}{command-ref-url-cli}[CLI command reference (rhoas)^]
+
+
+[id="ref-kafka-instance-settings_{context}"]
+== Kafka instance settings in {product-kafka}
+
+[role="_abstract"]
+You can edit the following Kafka instance settings in {product-long-kafka}.
+
+Connection re-authentication::
++
+--
+When a client connects to a Kafka instance, the session lasts for five minutes. 
+At that point, the client must reauthenticate to stay connected.
+Many Kafka clients automatically reauthenticate to remain connected,
+but some Kafka clients don't.
+
+If you use a Kafka client that doesn't support connection reauthentication,
+it will be disconnected when the five-minute session expires.
+To prevent the client from being disconnected every five minutes,
+disable the `Connection re-authentication` setting.
+
+Before disabling connection re-authentication,
+you should be aware of the security risks.
+If you disable connection re-authentication,
+and then a bad actor obtains credentials to your Kafka instance,
+they will be able to stay connected indefinitely.
+Deactivating the user account or service account will not close the connections that the bad actor has opened.
+In this scenario, you would need to add ACLs to prevent the unauthorized connections from performing any operations
+(see {base-url}{access-mgmt-url-kafka}[Managing account access in OpenShift Streams for Apache Kafka^]).
```

Comment:
A few comments on this definition:

- Avoid use of contractions
- Spell out _Access Control List_ in full when first introducing 'ACL'. Also, I'm not sure it needs to be _ACLs_ (plural) in this case? That is, I think you could define a single ACL to prevent bad actors from performing operations?
- Write this in the present tense. Avoid _will_, _would_, and _could_.
- We all understand the term "bad actor" but I still wonder if it borders on engineering jargon? Maybe we should consider a more generic phrase such as "malicious entity" or "malicious source"? At the very least, we probably need to define _bad actor_ (that is, an entity that poses a security risk) on first use.

Reply:
Just commenting here to follow this discussion regarding "bad actor". 😄 We want to use a similar message in the prompt in the UI, e.g.:
> There are potential risks associated with turning off re-authentication, where bad actors could stay connected indefinitely.

We can update the text in the UI to match whatever the final decision is here.

Reply:
Thanks for the comments, @jbyrne-redhat. I made some edits based on them. Also, I changed "bad actor" to "attacker".